Packages/CommonUI/CommonUI/Classes/View/MessageView/MessageBubleView.swift

```diff
@@ -69,38 +66,181 @@ public struct MessageBubbleView: View {
 
 // MARK: - Private view
 private extension MessageBubbleView {
+    
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/5abb0572a2378570e207060e71d026e6cf8ce71f/Packages/CommonUI/CommonUI/Classes/View/MessageView/MessageBubleView.swift#L69"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>MessageBubleView.swift:69</code></td>
    </tr>
  </tbody>
</table>